hera_cal/datacontainer.py

```diff
@@ -468,7 +469,7 @@ def get(self, key, val=None):
         '''Allows for getting values with fallback if not found. Default None.'''
         return (self[key] if key in self else val)
 
-    def select_or_expand_times(self, new_times, in_place=True, skip_bda_check=False):
+    def select_or_expand_times(self, new_times: Sequence[float] | None = None, in_place=True, skip_bda_check=False, *, indices: np.ndarray | None = None):
```

Comment:
I've added the docs. It's generally better to force things to be a kwarg in these kinds of cases, since then we can trust in future code that anyone calling the function has used a kwarg, so we can add new parameters without breaking the API.